### ISSUE TITLE
NullPointerException-s when 'component' word used in function comment in an interface

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#439]
The following two snippets will both cause `NullPointerException`s:

```
interface {
	/**
	 * component
	 */
	any function foo(required any bar);
}
```

and:

```
interface {
	/**
	 * component
	 */
	function foo(required any bar);
}
```

This happens only when:

* it's an interface
* the function has at least one argument
* the comment (or `@hint`) of the function has the word `component` in it

**Comment by ryaneberly** [comment]
Wow, that parsing isn't implemented.   I'm suprised it hasn't been mentioned before
https://github.com/cfparser/cfparser/issues/93

will be fixed in next cfparser release

**Comment by ryaneberly** [comment]
fixed in dev branch with cfparser 2.5.4
